docs/sphinx/ome-tiff/specification.rst

```diff
@@ -434,3 +438,96 @@ The master file containing the full OME-XML metadata should be either an
 OME-XML companion file with the extension :file:`.companion.ome` or a master
 OME-TIFF file containing the full metadata (see :ref:`multifile_samples` for
 representative samples).
+
+Pyramidal OME-TIFF
+------------------
+
+.. versionadded:: 6.0.0
+
+The OME-TIFF specification supports multi-resolution images or pyramidal
```

Comment:
`The OME-TIFF specification supports`.  This document *is* the specification, so this wording is redundant, and a bit confusing (is it referring to somewhere else?)